ports/stm32/eth.c

```diff
@@ -556,6 +633,13 @@ STATIC int eth_tx_buf_send(void) {
             | 1 << TX_DESCR_3_FD_Pos // first segment
             | 3 << TX_DESCR_3_CIC_Pos // enable all checksums inserted by hardware
     ;
+    #elif defined(STM32H5)
+    tx_descr->tdes3 =
```

Comment:
Can this same code be combined with above?

Reply:
Done